odoo/addons/base/views/ir_qweb_widget_templates.xml

```diff
@@ -33,6 +33,20 @@
                 <span>
                     <div>
                         <span itemprop="addressLocality" t-esc="city"/>,
+                        <t t-if="object.state_id and object.country_id.state_required">
+                            <span itemprop="addressState" t-esc="object.state_id.code"/>,
+                        </t>
+                        <t t-if="country_id"><span itemprop="addressCountry" t-esc="country_id"/></t>
+                    </div>
+                </span>
+            </div>
+            <div t-elif="not city and country_id and 'country_id' in fields" t-attf-class="d-flex align-items-baseline gap-1">
```

Comment:
Seems it is already implied country_id is in fields as we do "object.country_id.name_position" above